packages/core/src/schema/models/workspaces.ts

```diff
@@ -1,14 +1,41 @@
-import { bigserial, text, varchar } from 'drizzle-orm/pg-core'
+import { bigint, bigserial, index, text, varchar } from 'drizzle-orm/pg-core'
 
+import { SubscriptionPlan } from '../../plans'
 import { latitudeSchema } from '../db-schema'
 import { users } from '../models/users'
 import { timestamps } from '../schemaHelpers'
 
 export const workspaces = latitudeSchema.table('workspaces', {
   id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
   name: varchar('name', { length: 256 }).notNull(),
+  currentSubscriptionId: bigint('current_subscription_id', {
+    mode: 'number',
+  }).references(() => subscriptions.id),
   creatorId: text('creator_id').references(() => users.id, {
     onDelete: 'set null',
   }),
   ...timestamps(),
 })
```

Comment:
We should add the time when the subscription changed, so that we can calculate when credits are renewed based on it.

Reply:
I don't understand this comment 😅

Reply:
The credit usage is calculated as the total of `documentLogs` and `evaluationResults` created in the workspace since a certain date.

This date is currently determined by the workspace creation date. For example, if a user created their workspace on July 6th, it will be renewed on the 6th of every month thereafter.

However, when a subscription plan is added, we need to calculate the date from the last time the user changed their subscription plan.

Reply:
oh i see

Reply:
That would be the subscription creation date